tests/foreman/cli/test_contentview.py

```diff
@@ -4662,24 +4659,20 @@ class TestContentViewFileRepo:
     arbitrary files
     """
 
-    def make_file_repository_upload_contents(self, module_org, module_product, options=None):
+    def make_file_repository_upload_contents(
+        self, module_org, module_product, sat=None, options=None
```

Comment:
Consider using `satellite` here so we stay consistent with kwarg naming for satellite/capsule instances.

Reply:
Since this function is defined within the test module, I think that the `satellite` instance can be required here, since all tests can pass `default_sat` to it.  Removes a use of `ssh.get_client`  that we'll want to remove one day anyway.

That or move it to the Capsule class as an instance method, since I actually don't see any references to `self` made within the function.

Reply:
I've made satellite required.